### ISSUE TITLE
Translators are reporting incorrect percentages in the report

### ISSUE THREAD

**Issue by jbphet** [phetsims/rosetta#421]
@solaolateju sent me a Slack message this morning where he said that translators are saying that they are seeing incorrect values for the percentage of strings that are translated.  The translators are saying that the utility is reporting less than 100% when all of the strings have been translated.

As an example, he Kikuyu translator (locale = `ki`) says he/she is seeing this in greenhouse-effect, build-a-molecule, and acid-base-solutions.

**Comment by jbphet** [comment]
I can see what they are talking about.  If I look at Kikuyu for acid-base-solutions I see this:

![image](https://github.com/phetsims/rosetta/assets/3082831/1c8a73b3-54ff-4b56-8cd2-86721339ba00)

But when I load the translation page for the sim, all but one string is translated.  It also lists the sim as "not yet published in Kikuyu", and I don't see it on the website.

@solaolateju - Can you please ask the translator if they have published the sim by pressing the "Publish" button?  Perhaps they are using the translation a little differently or something.  If they haven't, please have them do so.

**Comment by RVieyra** [comment]
We have known for some time that the % Translated Strings is not accurate.

This does not prevent the publication.

And yes, our translators keep pressing "Publish" and it's just not showing up on the website.

We have noted this issue in Shona and Kikuyu.

**Comment by solaolateju** [comment]
Screenshot: Acid-base solutions
Locale: Kikuyu

Sorry @jbphet, this notification got drowned in all the GitHub notifications. 

Yes, they hit the publish button. I also tested with Kikuyu. 

When you hit the publish button, you receive the regular prompt before submission that shows all the strings are translated.
<img width="441" alt="image" src="https://github.com/phetsims/rosetta/assets/18397749/d7352ac9-80ae-4fb5-b6d5-449b3bd63f7a">


Followed by a confirmation of submission. 
<img width="436" alt="image" src="https://github.com/phetsims/rosetta/assets/18397749/d137a3ef-fb53-4cd7-8cbf-889838a9432b">

However, the submitted sim stays in the list of "Sims not yet published" and cannot be viewed on the website.

**Comment by jbphet** [comment]
I just checked the main PhET website and found that the Kikuyu translation of Acid Base solutions is present.  I looked at the history of the main translation strings file (`acid-base-solutions-strings_ki.json`), and it looks like the translator attempted to publish on 7/13/2023 and 7/20/2023.  Perhaps the first publication attempt failed for some reason and the second succeeded.  The sim is now listed as being published in Kikuyu, here is a screenshot from the translation utility:

![image](https://github.com/phetsims/rosetta/assets/3082831/7c151e6c-0e44-44e5-8568-2b4d35a3677d)

I think this particular sim/locale is now looking correct, and I'll pursue some of the other issues with mis-reported stats.

**Comment by jbphet** [comment]
@solaolateju - I just looked at the stats for the Kikuyu translations of greenhouse-effect, build-a-molecule, and acid-base-solutions, which were identified at one point as being incorrect.  They all look correct to me.  Can you please check and/or have the Kikuyu translator check and see if these issue appear to be resolved?

**Comment by jbphet** [comment]
@RVieyra said in a [previous comment](https://github.com/phetsims/rosetta/issues/421#issuecomment-1646033127):

> We have known for some time that the % Translated Strings is not accurate.

I've tracked down and, I believe, fixed a couple of issue that were causing inaccuracies in the percentage values in the translation reports.  I spot tested several simulations in the Kikuyu and Shona locales, and the information was accurate. 
 @solaolateju and @RVieyra - can you please check this and let me know if the problem appears to have been addressed?

**Comment by oliver-phet** [comment]
Just adding to this issue that Center and Variability is displaying 216% (in the case of Polish -- fully translated)
<img width="969" alt="image" src="https://github.com/phetsims/rosetta/assets/5090275/3d698141-66fa-4c84-abe4-fb425d859778">
Spanish is similar:
<img width="854" alt="image" src="https://github.com/phetsims/rosetta/assets/5090275/b4b9f021-f333-4db8-b33b-1653f6aee07c">

**Comment by solaolateju** [comment]
@jbphet we are having new instances of % completion status issue. Please take a look.

**Comment by jbphet** [comment]
@kathy-phet has let me know that fixing this isn't a priority at the moment.  I'll leave it open and will check during planning meetings in the future, but as of now it is unlikely to get much of my attention anytime soon.

**Comment by RVieyra** [comment]
Understood. We will just let the translators know to ignore the value for now.

**Comment by jbphet** [comment]
[Above](https://github.com/phetsims/rosetta/issues/421#issuecomment-2032174730), @solaolateju said:

> @jbphet we are having new instances of % completion status issue. Please take a look.

I've been allocated a bit of time in the current iteration to work on translation, so I am, indeed, taking a look.  I'm testing using the currently live version at `https://phet.colorado.edu/translate`.  I'm looking at the percentage values for Center & Variability for the Polish (pl) and Spanish (es) locales because @oliver-phet [reported some specific issues with these](https://github.com/phetsims/rosetta/issues/421#issuecomment-2021621804), but they seem fine.  Here are some screenshots:

Polish C&V:

![image](https://github.com/user-attachments/assets/b1611c10-617f-46af-9184-1aa2cc73c02b)

Spanish C&V:

![image](https://github.com/user-attachments/assets/2009b140-7d76-43a4-abbf-9247fe7a5291)

The full translation report page for Polish shows everything at 100%.  Diligent translator I guess.

<details>

<summary>Screen capture of full Polish page</summary>

![screencapture-phet-colorado-edu-translate-pl-2024-08-23-16_41_54](https://github.com/user-attachments/assets/af0d3e1e-0331-4636-8a31-41a7f29c6863)

</details>

The Spanish (es) translation report page shows some sims that are not fully translated, and I spot checked some of these (Diffusion, Gas Properties, and My Solar System), and the percentage values appear correct.

<details>
<summary>Screen capture of full Spanish (es) page</summary>

![screencapture-phet-colorado-edu-translate-es-2024-08-23-16_54_13](https://github.com/user-attachments/assets/eee19848-f41f-448c-ac2c-8c25aadbe74d)

</details>

The translation report page for Kikuyu shows some sims at 100% and some below.  I spot checked several sims (Acid Base Solutions, My Solar System, and Vector Addition), and the values seem correct.

<details>
<summary>Screen capture of full Kikuyu(ki) page</summary>

![screencapture-phet-colorado-edu-translate-ki-2024-08-23-17_17_27](https://github.com/user-attachments/assets/043409ed-d34b-403f-8e0d-8aca8d379213)

</details>

The bottom line here is that all my testing of the currently live version shows the percentage values as being accurate.  This means that I'll need a bit more information from @oliver-phet, @solaolateju, and/or @RVieyra to figure out why problems are being reported.  Are any of you able to come up with example cases where the percentages are incorrect?  If so, please provide me with screenshots and as much information as possible with regard to locale, sim, translator, and when they were seeing the problem (local time zone is fine as long as you can tell me where they are so I can convert to GMT.  I need this to be able to check Rosetta's logs).  Otherwise I'm afraid there isn't much I can do.

**Comment by solaolateju** [comment]
@jbphet Thanks for taking a look at this issue again. 
I also took a look and everything at first glance all is looking OK now. 
Let me check with the network tomorrow and come back after a final check with them.

**Comment by jbphet** [comment]
I just spoke with @solaolateju over Slack, and he says that it's safe to close this issue now - things seem to be working correctly.
